config/settings/local.py

```diff
@@ -123,6 +123,16 @@
 CONNECTED_APP_CLIENT_SECRET = env("CONNECTED_APP_CLIENT_SECRET")
 CONNECTED_APP_CALLBACK_URL = env("CONNECTED_APP_CALLBACK_URL")
 
+METACI_APPS = {"local-app"}
+
+AUTOSCALERS = {
+    "local-app": {
+        "app_name": "local-app",
+        "worker_type": "worker",
+        "queues": ["default", "medium", "high"],
```

Comment:
For running locally we should probably include the robot queue here (i.e. run it in the same workers locally)